test_schema.py

```diff
@@ -240,6 +240,22 @@ def test_dict_subtypes():
     # is dropped!
 
 
+def test_dict_key_error():
+    try:
+        Schema({'k': int}).validate({'k': 'x'})
+    except SchemaError as e:
+        assert e.code == "Key 'k' error:\n'x' should be instance of 'int'"
+    try:
+        Schema({'k': {'k2': int}}).validate({'k': {'k2': 'x'}})
+    except SchemaError as e:
+        code = "Key 'k' error:\nKey 'k2' error:\n'x' should be instance of 'int'"
```

Comment:
@skorokithakis I think this PEP8 looks ok.
https://travis-ci.org/keleshev/schema/jobs/183802776
So please tell me a problem if it still exists. I want to fix it